### ISSUE TITLE
Is this still an active project?

### ISSUE THREAD

**Issue by outermeasure** [noiszy/noiszy#4]
Context: "The Social Dilemma" documentary on Netflix.

**Comment by almera-vs** [comment]
I don't think so. There are no new commits and their website wasn't updated since 2017. Or maybe this plugin is 100% done but I highly doubt it.

Anyway if this project would be more popular then maybe something interesting would happen.

**Comment by muhlinux** [comment]
check out this fork https://github.com/2br-2b/datafaux

**Comment by matthew-james-brewer** [comment]
datafaux isn't maintained either (anymore)
